docs/CONFIG-PROPERTIES.md

```diff
@@ -35,6 +35,7 @@
 | maintenance.mode | "enabled" or "disabled" | "none" | don't run applications etc |
 | force.fallback.counter | integer | 0 | forces fallback to other image if counter is changed |
 | newlog.allow.fastupload | boolean | false | allow faster upload gzip logfiles to controller |
+| memory.apps.ignore.check | boolean | false | Ignore memory usage check for Apps|
```

Comment:
I must be missing something obvious -- but I don't see this used anywhere. What am I missing?

Reply:
Good catch; there should be some if !IgnoreMemoryCheckForApps checks before checking the space.

Reply:
@rvs Thanks for pointing it. Missed it out. Will add it.